streampark-storage/src/main/java/org/apache/streampark/storage/StorageService.java

```diff
@@ -17,8 +17,11 @@
 
 package org.apache.streampark.storage;
 
+/**
+ * StorageService will be used as artifacts fetcher in pod template, so it should rely on other modules.
+ */
 public interface StorageService {
```

Comment:
StorageService just be used as artifacts fetcher in pod template, right?

If yes, the class name is unreasonable. The StorageService looks like a generic storage service.

Reply:
> StorageService just be used as artifacts fetcher in pod template, right?
> 
> If yes, the class name is unreasonable. The StorageService looks like a generic storage service.

@1996fanrui  No, this will be used to uploading job-related resources to remote storage system.

Reply:
Thanks for you quick reply.

If no, I have some questions:

1. why did you add the comments for  `StorageService`? 

> StorageService will be used as artifacts fetcher in pod template, so it should rely on other modules.

2. Whether the localFilePath parameter of 2 methods are reasonable? 

- I worry that the current interface definition only solves the current needs and will not be scalable in the future. 
- For example, will the distributed file system be supported in the future? Or just download the data into memory.
- Would it be more general to replace `String localFilePath` with `OutputStream/InputStream`?

3. Whether the objectPath of 2 methods are reasonable? 

- As I understand, ObjectStorage is just one type of StorageService, it shouldn't always be `String objectPath`, right?

Please correct me if there are any mistakes.